jekyll/_cci2/server/v4.3/installation/phase-1-prerequisites.adoc

```diff
@@ -555,6 +555,32 @@ aws iam create-role --role-name circleci-s3 --assume-role-policy-document file:/
   ]
 }
 ----
++
+NOTE: If you wish to store artifacts which are larger than 5GB, you will need to enable non-presigned URLs. To do this you will need your IRSA role to assume itself. Replace your trust policy above with the contents below.
```

Comment:
```suggestion
NOTE: If you wish to store artifacts which are larger than 5GB, you will need to disable presigned URLs. To do this you will need your IRSA role to assume itself. Replace your trust policy above with the contents below.
```
`non-presigned URLs` doesn't really correctly describe how the non-presigned mode works.